main.tf

```
@@ -191,6 +202,7 @@ module "vpc" {
 
   enable_nat_gateway = true
   single_nat_gateway = true
+  enable_dns_hostnames = true
```

I think this should be conditional, like this:

```suggestion
  enable_dns_hostnames = var.enable_ephemeral_storage
```

---

`enable_ephemeral_storage` actually defaults to false, so I inversed the bool and converted this to a conditional as you suggested in [b6d233e](https://github.com/terraform-aws-modules/terraform-aws-atlantis/pull/247/commits/b6d233e66dffd92b13d3111ba66ba874b764a37e)